portingdb/templates/index.html

```diff
@@ -24,42 +64,68 @@ <h1>
         <div class="col-md-12">
             {{ progress_summary(status_summary, total_pkg_count) }}
         </div>
+        <div class="col-md-12">
+            <div class="main-summary">
+            <div class="summary-sentence">
+                <span class="big-percentage"
+                    style="color: {{ statuses['py3-only'].color }};"
+                >
+                    {{ (the_score * 100) | int }}%
```

Comment:
round? at least the title now says 44.9 %, but the page says 44 %.

Reply:
Alright, let me fit that in